packages/libs/react-components/tsconfig.test.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
nice! glad this works! 

Some things that still need to change: 
- I think it would be better to name this tsconfig.esm.json since this transpiles the lib for esm
- We need to update the package.json `build` script from 
```"build": "tsc & tsc -p ./tsconfig.cjs.json",```
to
```"build": "tsc -p ./tsconfig.esm.json & tsc -p ./tsconfig.cjs.json",```

Reply:
Also the `tsconfig.cjs.json` will need to extend the `tsconfig.esm.json` since it was previously extending that tsconfig and overwriting certain values

Reply:
done